crates/holochain/src/conductor/ribosome_store.rs

```diff
@@ -37,28 +37,40 @@ impl RibosomeStore {
 
     /// List all DNA hashes in the store.
     #[cfg_attr(feature = "instrument", tracing::instrument(skip(self)))]
-    pub fn list(&self) -> Vec<DnaHash> {
-        self.ribosomes.keys().cloned().collect()
+    pub fn list_dna_hashes(&self) -> Vec<DnaHash> {
```

Comment:
Let's call out what we list. Also, this will for now return duplicate dna hashes in case multiple cells with the same dna hash are installed. I'm happy to change that if we think we should.

Reply:
I don't see the value of duplicates in the list if we could return a HashSet here?

Reply:
Good point, let's do that.

Reply:
https://github.com/holochain/holochain/pull/5189/commits/d0e21a0aa8677e55bb2fb22d40300013969a4a53 - I kept the admin interface response as a Vec. Do you think I should change that as well?